source/slang/slang-ast-support-types.h

```diff
@@ -221,6 +221,25 @@ FIDDLE() namespace Slang
 
     char const* getGLSLNameForImageFormat(ImageFormat format);
 
+    /// Enum for known built-in function names to replace string-based comparisons
+    enum class KnownBuiltinDeclName : uint32_t
+    {
+        GeometryStreamAppend,
+        GeometryStreamRestart,
+        GetAttributeAtVertex,
+        DispatchMesh,
+        saturated_cooperation,
+        saturated_cooperation_using,
+        IDifferentiable,
+        IDifferentiablePtr,
+        IDifferentiablePtrType,
```

Comment:
We are not using the `IDifferentiablePtrType` case anywhere in compiler code or the core library code, can we remove this?